src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking/single-button-audio-player/single-button-audio-player.component.scss

```diff
@@ -25,6 +24,6 @@ button:hover {
   cursor: pointer;
 }
 
-mat-icon {
-  color: $buttonColor;
+button:focus-visible {
+  outline: none;
```

Comment:
The outline (use Tab) is cut off, and I haven't been able to figure out how to style it such that it a) isn't cut off, and b) doesn't conflict with the radial progress.